.github/workflows/release-drafter-ca.yml

```diff
@@ -0,0 +1,147 @@
+name: Schedule a release and tag image for the cotnainer app
+on:
+  workflow_dispatch:
+  schedule:
+     - cron: "0 0 * * 3"
+
+env:
+  RELEASEPREFIX: auditlogca-
+  VERSIONPREFIX: v
+  REGISTRY: ghcr.io
+  REPOSITORY: altinn/altinn-auth-audit-log
+
+jobs:     
+  getlatestrelease:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: write
+      issues: write
+      packages: write
+    
+    steps:
+      - name: Get latest version for function app
+        uses: oprypin/find-latest-tag@v1
+        continue-on-error: true
+        with:
+          repository: ${{ env.REPOSITORY }}
+          releases-only: true
+          prefix: ${{ env.RELEASEPREFIX }}${{ env.VERSIONPREFIX }}  # Other kinds of tags are irrelevant;
+        id: auditlogca_version
```

Comment:
Please have a newline between steps here. It makes it much less likely to overlook in review.